### DISCUSSION TITLE
Approved but not received benefits yet

### DISCUSSION THREAD

**Discussion by hemantchettri** [community/community#135203] [GitHub Education]
### Hi everyone,

I have not received your education benefits and it's already been more than 5 days. I have already tried to login with my college domain mail id which I have shared to github while activating my student benefit.

![image](https://github.com/user-attachments/assets/4851a2cb-9b43-46c7-9d87-9df8e3445b89)

**Comment by everforyou** [comment]
Hi, please ensure that you have checked your email (including spam/junk folders) for any communication regarding the activation of your benefits.
If not get in there, you can reach out to the support team or the relevant department that handles education benefits. Provide them with your details and the issue you are facing.

**Comment by davevad93** [comment]
Hi @hemantchettri ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @hemantchettri was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Tehmeena0** [comment]
It can sometimes take more than 5 days for verification. Make sure the email you provided matches your college domain exactly and check your spam folder. If everything seems correct, try reaching out

**Comment by Nipun-Sulochana** [comment]
It can take up to a couple of weeks for education benefits to activate. Ensure you've checked all email folders (including spam) for communication. Confirm that the email you used matches your college domain exactly. If you still have issues, contact the support team with your details.
